src/config/mozambique/layers.json

```diff
@@ -21,9 +22,9 @@
     "type": "boundary",
     "path": "../data/mozambique/moz_admin1_boundaries.json",
     "opacity": 0.8,
-    "admin_code": "ADM2_PCODE",
-    "admin_level_names": ["ADM1_PT", "ADM2_PT"],
-    "admin_level_local_names": ["ADM1_PT", "ADM2_PT"],
+    "admin_code": "ADM1_PCODE",
```

Comment:
Is ok to display only admin1 level data? 

![image](https://user-images.githubusercontent.com/3285923/146954658-a761ef0d-b35a-48e0-a13b-280bc7a488a1.png)

Reply:
@JorgeMartinezG my mistake. Correcting it.

Reply:
why do we have changes in this config file?

Reply:
@erwagasore it's still happening. But if you toggle the admin level 1 test data and then back, it works as expected.
So maybe something to look at there.